lib/src/bloc_select.dart

```diff
@@ -13,8 +15,9 @@ class SelectBloc {
   /// Create a select bloc with the specified options. The select
   /// bloc will fire a query on creation
   SelectBloc(
-      {@required this.table,
-      @required this.database,
+      {@required this.database,
+      this.query,
+      this.table,
```

Comment:
As the `table` argument is not required anymore it could be good to add a check and throw an `ArgumentError` with an informative message if both `table` and `query` are null

Reply:
You're absolutely right, some check is required. 
Perhaps I was too hasty with removing the `assert`... Maybe I should put it back like this: 
```
assert(query.isNotEmpty || table.isNotEmpty)
```

What do you think?